nixos/modules/services/web-servers/nginx/default.nix

```diff
@@ -189,20 +189,15 @@ let
 
       ${optionalString (cfg.recommendedGzipSettings) ''
         gzip on;
-        gzip_proxied any;
-        gzip_comp_level 5;
-        gzip_types
-          application/atom+xml
-          application/javascript
-          application/json
-          application/xml
-          application/xml+rss
-          image/svg+xml
-          text/css
-          text/javascript
-          text/plain
-          text/xml;
+        gzip_static on;
         gzip_vary on;
+        gzip_disable "MSIE [1-6]\.";
+        gzip_http_version 1.1;
+        gzip_comp_level 5;
+        gzip_min_length 256;
+        gzip_proxied expired no-cache no-store private auth;
+        gzip_types ${lib.concatStringsSep " " compressMimeTypes};
+        gzip_buffers 32 8k;
```

Comment:
Same question here.

Reply:
This is more or less the default https://nginx.org/en/docs/http/ngx_http_gzip_module.html#gzip_buffers and I would just leave it to nginx to use the auto value

Reply:
Here the buffer is increased from 4k to 8k.

Reply:
I doubt that this improves things and it could actually make things worse. The setting is depending on the page size of the OS and that will have memory alignment reasons. I tried to find a resource which explains why changing this would be a good idea and only found guides which just changed the setting with no explanation. 

Since all the other settings make sense to me and are so recommended in nginx docs, I am just going to drop this and then merge it.

Reply:
```suggestion
```